src/Makefile.test.include

```diff
@@ -46,7 +46,7 @@ BITCOIN_TESTS =\
   test/compress_tests.cpp \
   test/crypto_tests.cpp \
   test/cuckoocache_tests.cpp \
-  test/DoS_tests.cpp \
+  test/denialofservice_tests.cpp \
```

Comment:
Why expand this?

Reply:
`dos` could mean DOS or DoS :-)

Reply:
While "DoS" is clearly "Denial of Service" (and not "Disk Operating System {DOS}"), I find that it's generally better to avoid using acronyms in order to simplify future code reviews, inspections, etc.